k8s-multus/scripts/node01.sh

```diff
@@ -16,4 +15,22 @@ kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /etc/kubernetes/flanne
 kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /etc/kubernetes/macvlan-conf.yml
 kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /etc/kubernetes/ptp-conf.yml
 kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /etc/kubernetes/bridge-conf.yml
-kubectl --kubeconfig=/etc/kubernetes/admin.conf taint nodes node01 node-role.kubernetes.io/master:NoSchedule- 
\ No newline at end of file
+kubectl --kubeconfig=/etc/kubernetes/admin.conf taint nodes node01 node-role.kubernetes.io/master:NoSchedule-
+
+# Wait for api server to be up.
+kubectl --kubeconfig=/etc/kubernetes/admin.conf get nodes --no-headers
+kubectl_rc=$?
+retry_counter=0
+while [[ $retry_counter -lt 20 && $kubectl_rc -ne 0 ]]; do
+    sleep 10
+    echo "Waiting for api server to be available..."
+    kubectl --kubeconfig=/etc/kubernetes/admin.conf get nodes --no-headers
+    kubectl_rc=$?
+    retry_counter=$((retry_counter + 1))
+done
+kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /tmp/local-volume.yaml
+
+# create local block device, backed by raw cirros disk image (see also provision.sh)
+LOOP_DEVICE=`losetup --find --show /mnt/local-storage/cirros.img.raw`
```

Comment:
Can it be part of https://github.com/kubevirt/kubevirt/tree/master/images/disks-images-provider?

Reply:
sounds like a good idea, thx :)

Reply:
got it working with disks-images-provider, will remove this stuff from CI in a minute